src/features/campaigns/components/CampaignActionButtons.tsx

```diff
@@ -79,6 +84,24 @@ const CampaignActionButtons: React.FunctionComponent<
       },
     });
   };
+
+  const handleCreateEvent = () => {
+    const defaultStart = new Date();
+    defaultStart.setDate(defaultStart.getDate() + 1);
```

Comment:
I think the start time should be rounded to the nearest hour (or, if that's easier, just remove minutes and seconds so that it's floored down to the start of the hour).

Reply:
I agree! thanks for the feedback!